### DISCUSSION TITLE
Service mirror Namespace

### DISCUSSION THREAD

**Discussion by srreddy2020** [linkerd/linkerd2#10998] [General]
Hello,
When we label the service like below,will it create the namespace in other crosscluster? or do we need to create namespace manually?
    mirror.linkerd.io/namespaceus.exported: "true"
    mirror.linkerd.io/namespaceeu.exported: "true"

**Comment by alpeb** [comment]
Hi, the namespace has to be created beforehand, otherwise the mirrored services won't get created. The reasoning here is that it's a less intrusive behavior and to avoid having to grant namespace creation RBAC to the service mirror controller.

**Reply by srreddy2020** [reply]
Thank you for the quick reply @alpeb
